contributions/api/models/reviewer.py

```diff
@@ -0,0 +1,41 @@
+#  Copyright 2021 Board of Trustees of the University of Illinois.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import contributions.api.utils.datasetutils as datasetutils
+
+class Reviewer:
+    def __init__(self, injson):
+        self.name = None
+        self.githubUsername = None
+        self.dateCreated = None
+
+        self, restjson = datasetutils.update_reviwer_dataset_from_json(self, injson)
```

Comment:
It looks like here `self` needs to be fixed, right? Also, I'm getting an error when doing a POST.
```
contributions/api/models/reviewer.py", line 23, in __init__
    self.reviewer_dataset, restjson = datasetutils.update_reviwer_dataset_from_json(self, injson)
AttributeError: module 'contributions.api.utils.datasetutils' has no attribute 'update_reviwer_dataset_from_json'
```

Reply:
I believe that it is related to the one that I fixed for packaging. Dang, it is problematic. Please give me some time to fix and test one more time. Sorry about that.